models/zeroex/celo/zeroex_celo_api_fills_deduped.sql

```diff
@@ -0,0 +1,102 @@
+{{  config(
+        tags=['dunesql'],
+        alias = alias('api_fills_celo_deduped'),
+        materialized='incremental',
+        partition_by = ['block_date'],
```

Comment:
We can add a new field `block_month` to use in partition:
```suggestion
        partition_by = ['block_month'],
```